frontend/javascripts/admin/voxelytics/workflow_list_view.tsx

```diff
@@ -86,7 +87,13 @@ export default function WorkflowListView() {
 
   usePolling(loadData, VX_POLLING_INTERVAL);
 
-  const renderRuns = useMemo(
+  const getUserDisplayName = (run: VoxelyticsWorkflowListingRun) => {
+    return run.userFirstName != null || run.userLastName != null
+      ? [run.userFirstName, run.userLastName].join(" ")
```

Comment:
```suggestion
      ? [run.userFirstName, run.userLastName].join(" ").trim()
```

because `[null, "hi"].join(" ")` evaluates to `' hi'`.